vote/cnab/app/Makefile

```diff
@@ -0,0 +1,31 @@
+CNAB_ACTION ?= status
+CNAB_INSTALLATION_NAME ?= vote
+
+CNAB_P_HELM_OPTIONS ?= --dry-run --debug
```

Comment:
I wonder if this env var should be empty by default so that it actually attempts chart installation?